packages/edit-site/src/components/global-styles/shadow-panel.js

```diff
@@ -146,9 +146,9 @@ function ShadowPopoverContainer( { shadow, onShadowChange } ) {
 function ShadowPresets( { presets, activeShadow, onSelect } ) {
 	return ! presets ? null : (
 		<Grid columns={ 6 } gap={ 0 } align="center" justify="center">
-			{ presets.map( ( { name, shadow }, i ) => (
+			{ presets.map( ( { name, slug, shadow } ) => (
 				<ShadowIndicator
-					key={ i }
+					key={ slug }
```

Comment:
This change isn't related to the PR, but we should only use `index` as a `key` as a last resort.

Happy to extract it into a new RP, if we want a separate entry in the git history.

Reply:
That's fine :)